onvm/go.sh

```diff
@@ -46,21 +86,7 @@ then
     exit 1
 fi
 
-shift 3
-
-if [ -z "$nf_cores" ]
-then
-    usage
-fi
-
-ports_detected=$("$RTE_SDK"/usertools/dpdk-devbind.py --status-dev net | sed '/Network devices using kernel driver/q' | grep -c "drv")
-if [[ $ports_detected -lt $ports ]]
-then
-    echo "Error: Invalid port mask. Insufficient NICs bound."
-    exit 1
-fi
-
-while getopts "a:r:d:s:t:l:p:z:cv" opt; do
+while getopts "a:r:d:s:t:l:p:z:cvm:k:n:" opt; do
```

Comment:
Probably I'm rusty on getopts, but wouldn't we want a `:` in between `cv` and `m`?

Reply:
I could be wrong on this, but my thought process was the following: the `:` after the option name denotes that there's an `OPTARG` for the option. The script originally didn't have a `:` after the `cv`, so I assumed those did not have an `OPTARG`. Accordingly, since `m`, `k`, and `n` have an `OPTARG` for each of them, they need the colon after them. Let me know if I'm wrong here, it's totally possible.